index.js

```diff
@@ -66,6 +66,7 @@ module.exports = exports = function fetch(url, opts = {}) {
 
         result.status = res.statusCode
         result.redirected = redirects > 0
+        if (result.status > 199 && result.status < 300) result.ok = true
```

Comment:
```suggestion
        if (result.status >= 200 && result.status <= 299) result.ok = true
```

That aligns with the specification text, which defines an "ok status" as:

> An ok status is a status in the range 200 to 299, inclusive.

https://fetch.spec.whatwg.org/#ok-status

Reply:
Let's actually make it a getter, there's no need for a property.